codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyHttpDecoder.java

```diff
@@ -346,7 +349,8 @@ protected void decode(ChannelHandlerContext ctx, SpdyFrame msg, List<Object> out
         }
     }
 
-    private static FullHttpRequest createHttpRequest(int spdyVersion, SpdyHeadersFrame requestFrame)
+    private static FullHttpRequest createHttpRequest(int spdyVersion, SpdyHeadersFrame requestFrame,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "spdyVersion". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1172)